pkg/sfu/buffer/rtpstats.go

```diff
@@ -56,25 +52,28 @@ type RTPDeltaInfo struct {
 }
 
 type Snapshot struct {
-	extStartSN          uint32
-	bytes               uint64
-	packetsDuplicate    uint32
-	bytesDuplicate      uint64
-	packetsPadding      uint32
-	bytesPadding        uint64
-	frames              uint32
-	nacks               uint32
-	plis                uint32
-	firs                uint32
-	maxJitter           float64
-	isJitterOverridden  bool
-	maxJitterOverridden float64
-	maxRtt              uint32
+	extStartSN            uint32
+	packetsDuplicate      uint32
+	bytesDuplicate        uint64
+	packetsLostOverridden uint32
+	nacks                 uint32
+	plis                  uint32
+	firs                  uint32
+	maxRtt                uint32
+	maxJitter             float64
+	maxJitterOverridden   float64
+}
+
+type SnInfo struct {
+	pktSize     uint16
+	payloadSize uint16
+	marker      bool
```

Comment:
This is the new struct to maintain packet level info which can be used to check sequence numbers in an interval.